cmd/cluster-image-registry-operator/main.go

```diff
@@ -22,6 +25,8 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
+var logLevel = flag.String("loglevel", "info", "sets the sensitivity of logging output.")
```

Comment:
`log-level` please, so we're consistent with everywhere else.

Reply:
oc cli uses `loglevel`: https://github.com/openshift/origin/blob/e004e6513ec755a7a106dc97516703847e470d0b/pkg/cmd/flagtypes/glog.go#L16

Reply:
hm. i swear i saw --log-level elsewhere, but ok.

Reply:
installer and cluster-operator use `--log-level`.

Reply:
that must be where i saw it. anyway --loglevel is fine since origin is using it, sorry for the noise.